data/geany.glade

```diff
@@ -1563,6 +1564,29 @@
                                               </packing>
                                             </child>
                                             <child>
+                                              <object class="GtkAlignment" id="alignment15">
+                                                <property name="visible">True</property>
+                                                <property name="can_focus">False</property>
+                                                <property name="left_padding">12</property>
+                                                <child>
+                                                  <object class="GtkCheckButton" id="check_symbols_sort_by_appearance">
+                                                    <property name="label" translatable="yes">Sort by appearance</property>
+                                                    <property name="visible">True</property>
+                                                    <property name="can_focus">True</property>
+                                                    <property name="receives_default">False</property>
+                                                    <property name="tooltip_text" translatable="yes">Default to sort by appearance instead of name</property>
```

Comment:
What about rather something like "Sort symbols by appearance instead of name by default"?  sounds more englishy to me, but I might be wrong